build.cmd

```diff
@@ -380,6 +380,36 @@ for /f "tokens=*" %%s in ('%DotNetCli% msbuild "%OptDataProjectFilePath%" /t:Dum
     set __IbcOptDataVersion=%%s
 )
 
+REM =========================================================================================
+REM ===
+REM === Generate source files for eventing
+REM ===
+REM =========================================================================================
+
+REM Find python and set it to the variable PYTHON
+echo import sys; print sys.executable | (python2.7 || python2 || py -2 || python) > %TEMP%\pythonlocation.txt 2> NUL
+set /p PYTHON=<%TEMP%\pythonlocation.txt
+if NOT DEFINED PYTHON (
+    echo %__MsgPrefix%Error: Could not find a python 2.7 installation
+    exit /b 1
+)
+
+set __IntermediatesIncDir=%__IntermediatesDir%\src\inc
+set __IntermediatesEventingDir=%__IntermediatesDir%\eventing
+
+if %__BuildNative% EQU 1 (
+
+    echo Laying out dynamically generated files consumed by the build system
+    echo Laying out dynamically generated Event test files and etmdummy stub functions
```

Comment:
Does etmdummy.h get used anywhere anymore?  If I remember correctly, it is not.

Reply:
It is used in gcenv.h, and eventtracebase.h, but I imagine we can get rid of it. I'll look into that

Reply:
I think you should leave the usage of it in gcenv.h - this is an abstraction layer for those who want to use the GC and provide their own EE implementation.  It can probably come out of eventtracebase.h, but because we need to keep it for gcenv.h, go ahead and leave it there too.  That way people can disable tracing at build time if they really want to without breaking things.

Reply:
Sorry for the false alarm.